### ISSUE TITLE
Update github issue template

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/eypd#605]
### Description 
To me (Amanda), the current issue template seems to be more suited for bugs than features. I'd like to add another section for feature requests. I don't have access to do this myself, so thought I'd make a new ticket so that we could have a bit of a conversation before changing the settings.

### User story
As a _[user type]_
I want _[thing you want]_
Because _[the reason you want it - should align with a site goal or problem that needs to be solved.]_

### Summary
_One to two line description of the feature_

### Acceptance criteria
- A bulleted list of details about
- How the feature should be implemented
- How the feature can be validated

### Mockup
_Supporting visual references for the feature. These could include wireframes and mockups of the actual site, links to other sites, gifs of transitions etc._

### Copy
_This section is optional. If any new text or edits is needed, add those details here_

### Related/Blocked by/Blocking
_Link any related issues by typing the issue number. Github will automatically link the issue. Add in the issue title after the number so that the list is easier to scan_

Related - #678 - Issue title copy and pasted in
Blocked by - #789 - Issue title copy and pasted in

**Comment by mandily-p** [comment]
Noted from December 6th standup - Brad would like to see us limit user roles to learner, organizer, and site admin with the ministry, data analyst, project manager etc all being grouped into the the site admin role and UX stories being from the perspective of one of the respective roles instead of independently.
